go.mod

```diff
@@ -34,13 +34,18 @@ replace go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => go.open
 
 replace github.com/jackc/pgx/v4 => github.com/jackc/pgx/v5 v5.5.4
 
+// These replace directives support the backlevel go version required by ODH build
+replace github.com/project-codeflare/appwrapper v0.21.1 => github.com/project-codeflare/appwrapper v0.21.2-0.20240712173553-5b007c947b37
+
+replace sigs.k8s.io/kueue v0.7.1 => github.com/opendatahub-io/kueue v0.7.0-odh-test
+
```

Comment:
Not thrilled with these replace directives, but we can't have the go.version at 1.22.2 and depend directly on the upstream kueue and appwrapper releases which are at 1.22.4 without breaking all the go tooling.